test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionController::TestCase
+
+  def setup
+    request.env["omniauth.auth"] = OmniAuth.config.mock_auth[:github]
+  end
+
+  test "should login" do
+    assert_difference('User.count', 1) do
+      get :create, provider: 'github'
+    end
+    assert_redirected_to users_path
+    assert_equal 'Signed in!', flash[:success]
+  end
```

Comment:
Please also add a test when the user is not new in which case the amount of users in the database shouldn't change.